src/test/java/uk/ac/ebi/eva/utils/JobParameterBuilder.java

```diff
@@ -0,0 +1,69 @@
+package uk.ac.ebi.eva.utils;
+
+import org.springframework.batch.core.JobParameter;
+import org.springframework.batch.core.JobParameters;
+import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
+
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JobParameterBuilder {
```

Comment:
It's to avoid that each one does a different solution to initialise jobParameters in the tests.

Didn't see that batch had a builder class already.